projects/plugins/migration/composer.json

```diff
@@ -59,9 +59,9 @@
 	"minimum-stability": "dev",
 	"prefer-stable": true,
 	"extra": {
-		"mirror-repo": "Automattic/jetpack-migration",
+		"mirror-repo": "Automattic/wpcom-migration",
```

Comment:
Think this needs to be changed by Jetpack Crew too as it references a repo.

Reply:
Yes, this is correct, once we have an approval on our team review, I'll communicate with Jetpack crew to help us makes the changes 👍